packages/gatsby/gatsby-node.js

```diff
@@ -46,10 +46,15 @@ exports.onCreateWebpackConfig = ({ plugins, getConfig, actions }) => {
           // Handle sentry-cli configuration errors when the user has not done it not to break
           // the build.
           errorHandler(err, invokeErr) {
-            const { message } = err;
+            const message = err.message && err.message.toLowerCase() || '';
             if (message.includes('organization slug is required') || message.includes('project slug is required')) {
```

Comment:
l: Do we maybe wanna add a similar error message right here as a quick win?

Reply:
As I wrote in the PR, there's an argument to be made that if these env vars are not specified, users don't want to upload source maps and hence there should be no warning. But let's just display a "info" level message here that we're not gonna upload source maps in case these variables aren't set.

Reply:
Ah sorry! I skipped over that. I think printing a message is good here since having source maps is such a productivity win - I think we wanna tell users about it :D